solidity/contracts/converter/BancorConverter.sol

```diff
@@ -536,9 +541,15 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
         validReserve(_reserveToken)
         returns (uint256, uint256)
     {
+        uint256 supply = token.totalSupply();
+        uint256 balance = reserveBalance(_reserveToken);
+
+        if (supply == _sellAmount)
+            return (balance, balance);

```

Comment:
same here - 2nd value should be 0, no?

Reply:
@yudilevi : No, it should be amount minus fee.

Reply:
@yudilevi : Damn, you're right!

Reply:
@yudilevi : Fixed.